torchinductor/overrides.py

```diff
@@ -163,3 +230,15 @@ def rand_like(x, **kwargs):
 
 
 replacements = {torch.nn.functional.dropout: lowmem_dropout, torch.rand_like: rand_like}
+
+computation_op = nn.Linear
```

Comment:
Currently, the computation operator only covers `Linear`. In the future, we also need to support `Conv` and `Matmul`. Should the `computation_op` be a list?

Reply:
Refactored the code to be more extensible to add support for Conv and Matmul in the future.